.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "trailingComma": "all",
+  "tabWidth": 2,
+  "semi": true,
+  "singleQuote": true,
+  "printWidth": 80,
```

Comment:
In my opinion, this is far too narrow. I'd like to propose a larger limit such as 120.

Reply:
120 sounds good to me